crypto/rand/rand_lib.c

```diff
@@ -46,7 +46,7 @@ void rand_read_tsc(RAND_poll_fn cb, void *arg)
     unsigned char c;
     int i;
 
-    for (i = 0; i < 10; i++) {
+    for (i = 0; i < TSC_READ_COUNT; i++) {
         c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
None of #3956 contradicts what I'm saying. It also effectively suggests that least significant bits of the TSC are not necessarily "most random" and that tight loops don't work.